### ISSUE TITLE
automatically update "provider map" dataset

### ISSUE THREAD

**Issue by hunterowens** [cal-itp/data-infra#28]
Currently, the [CA Mobility Marketplace](https://www.camobilitymarketplace.org/provider-map) has a provider map that was created out of a dataset that was given to the @thekaveman as a static CSV. 

We should modernize the process so the dataset is automatically updated and Kegan can use that

**Comment by thekaveman** [comment]
FYI that CSV file lives here: https://github.com/cal-itp/transit-stop/blob/master/metadata/providers/providers.csv

**Comment by hunterowens** [comment]
Discussion from today's meeting: 

Use a weekly GH action to 

1) Download the latest file from BQ - [docs](https://github.com/google-github-actions/setup-gcloud)
2) Automatically create a pull request - [docs](https://github.com/marketplace/actions/create-pull-request)

**Comment by hunterowens** [comment]
waiting on #118

**Comment by indexing** [comment]
@hunterowens I'm reviewing legacy Github issues for relevancy. Can we close this one?

**Comment by thekaveman** [comment]
@indexing we'll still need a solution for this for cal-itp/mobility-marketplace#440 and it will likely be driven out of the warehouse. 

cc @mrose914.

**Comment by hunterowens** [comment]
New issue in Data Analysis  is good! With Jarv.us contract wrapping up it will either need to be CT or Digital Services to self serve out of warehouse, but should be doable.
